src/main/groovy/org/pih/warehouse/importer/CSVUtils.groovy

```diff
@@ -200,4 +201,30 @@ class CSVUtils {
         detector.dataEnd();
         return detector.getDetectedCharset() ?: 'MacRoman';
     }
+
+    static List<Map<String, String>> csvToObjects(String csvString) {
+        CSVParser parser = CSVParser.parse(
+                csvString,
+                CSVFormat.DEFAULT.withFirstRecordAsHeader().withTrim()
+        )
+
+        return parser.collect { record ->
+            record.toMap().collectEntries { k, v ->
+                [(toCamelCase(k)): v]
```

Comment:
Why do we want to modify the input data? What does it give us?

Reply:
The headers of CSV that we are using are like "Product Code", etc., so this is parsing it to "productCode" to work with the import method

Reply:
What if someone mistypes P**or**duct Code? Will it try to map to p**or**ductCode, which would later result in some potential issues? Or there will be an exception meaningful to the user, or will he be stuck trying to figure out why his file is not being imported?

Reply:
ok